# global-exception-handling
Here I handle exception using globalException Handler
